examples/flow_tracker/flow_tracker.c

```diff
@@ -291,22 +306,46 @@ callback_handler(__attribute__((unused)) struct onvm_nf_local_ctx *nf_local_ctx)
 static int
 packet_handler(struct rte_mbuf *pkt, struct onvm_pkt_meta *meta,
                __attribute__((unused)) struct onvm_nf_local_ctx *nf_local_ctx) {
+
         if (!onvm_pkt_is_ipv4(pkt)) {
                 meta->destination = state_info->destination;
                 meta->action = ONVM_NF_ACTION_TONF;
                 return 0;
         }
 
+	#if 0
+	printf("\n\n********************pkt*******************\n");	
+	onvm_pkt_print(pkt);
+	printf("********************end pkt*******************\n"); 
+	#endif
+
         if (table_lookup_entry(pkt, state_info) < 0) {
                 printf("Packet could not be identified or processed\n");
         }
-
         meta->destination = state_info->destination;
         meta->action = ONVM_NF_ACTION_TONF;
 
         return 0;
 }
 
+void
+nf_setup(struct onvm_nf_local_ctx *nf_local_ctx) {
+        printf("Hi boi %d\n", nf_local_ctx->nf->instance_id);
+
+	struct event_msg *msg = rte_zmalloc("ev msg", sizeof(struct event_msg), 0);
+        msg->type = RETRIEVE;
```

Comment:
indentation starting on this line doesn't match rest of file. Since we will want to release this code later, be sure you are being consistent with the coding styles we follow in the rest of ONVM.